test/net/sourceforge/kolmafia/session/ForbiddenManagerTest.java

```diff
@@ -10,23 +10,17 @@
 import net.sourceforge.kolmafia.preferences.Preferences;
 import net.sourceforge.kolmafia.request.MallPurchaseRequest;
 import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class ForbiddenManagerTest {
 
-  private static final String TESTUSERNAME = "ForbiddenManager";
-
-  @BeforeAll
-  public static void beforeAll() {
-    KoLCharacter.reset(TESTUSERNAME);
-    MallPurchaseRequest.reset();
-  }
+  private static final String TEST_USERNAME = "ForbiddenManager";
 
   @BeforeEach
   public void beforeEach() {
-    Preferences.reset(TESTUSERNAME);
+    KoLCharacter.reset(TEST_USERNAME);
+    MallPurchaseRequest.reset();
```

Comment:
Resetting the character also resets the preferences.  Resetting the MallPurchaseRequest keeps things from leaking since the request is reused.